app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -197,30 +197,31 @@ class BrowserTabViewModel(
         val browserShowing: Boolean = false,
         val isFullScreen: Boolean = false,
         val isDesktopBrowsingMode: Boolean = false,
-        val canChangeBrowsingMode: Boolean = true,
+        val canChangeBrowsingMode: Boolean = false,
         val showPrivacyGrade: Boolean = false,
         val showSearchIcon: Boolean = false,
         val showClearButton: Boolean = false,
         val showTabsButton: Boolean = true,
         val fireButton: HighlightableButton = HighlightableButton.Visible(),
         val showMenuButton: HighlightableButton = HighlightableButton.Visible(),
         val canSharePage: Boolean = false,
-        val canAddBookmarks: Boolean = false,
+        val canSaveSite: Boolean = false,
         val bookmark: SavedSite.Bookmark? = null,
         val addFavorite: HighlightableButton = HighlightableButton.Visible(enabled = false),
         val favorite: SavedSite.Favorite? = null,
         val canFireproofSite: Boolean = false,
         val isFireproofWebsite: Boolean = false,
         val canGoBack: Boolean = false,
         val canGoForward: Boolean = false,
-        val canWhitelist: Boolean = false,
-        val isWhitelisted: Boolean = false,
+        val canChangePrivacyProtection: Boolean = false,
+        val isPrivacyProtectionEnabled: Boolean = false,
         val canReportSite: Boolean = false,
         val addToHomeEnabled: Boolean = false,
         val addToHomeVisible: Boolean = false,
         val showDaxIcon: Boolean = false,
         val isEmailSignedIn: Boolean = false,
-        var previousAppLink: AppLink? = null
+        var previousAppLink: AppLink? = null,
+        val forceRenderingTicker: String = ""
```

Comment:
As far as I can see, this property is only used to force `browserViewState`, so its observers can react to it. If that's the case, why do we need to be this to a String?

I don't particularly like this solution, but we can't do much until we migrate this `LiveData` to `StateFlow`. At that point we can do something like `private var tickerChannel = MutableStateFlow(System.currentTimeMillis())` and combine the two `StateFlow`

Reply:
That's correct, it's also related to an specific scenario in the fragment:
- when device changes orientation the activity is not destroyed (see `onConfigurationChanged` comment)
- renderIfChanged logic in the fragment
- and, holding `lastSeen...ViewState`

 I guess you are proposing to use `long` instead? I'm fine with that as well 👍